src/components/Map/MapToolbar.js

```diff
@@ -21,28 +21,28 @@ export const MapToolbar = () => {
       <section className="state-heading">
         {!filteredStates && <span>United States of America</span>}
         <span>{filteredStates}</span>
-        {filteredStates && (
-          <a
+        {!!filteredStates && (
+          <button
```

Comment:
This breaks the styling

<img width="390" alt="Screenshot 2024-02-07 at 3 19 25 PM" src="https://github.com/cfpb/ccdb5-ui/assets/1038306/bc835efb-5693-4668-9064-166d1e03a066">


It should look like
<img width="392" alt="Screenshot 2024-02-07 at 3 20 19 PM" src="https://github.com/cfpb/ccdb5-ui/assets/1038306/6fbe94d1-b456-4984-8763-343e5f367b0c">

Go to the map tab and click some state tiles to see where the links should appear.

Reply:
Ok, these are fixed.